### PR TITLE
Fix nobr element serialization/deserialization

### PR DESCRIPTION
**Pull request by roadlittledawn**
## Description

Adds handlers to serialize / deserialize an element that somehow found its way in some docs. It caused errors when translation workflows ran to send content to translation / fetch and deserialize. I also removed those elements because the [nobr element is deprecated should not be used](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nobr).

PR also includes prettier applied formatting.

## Related issues / PRs
Related to #4094

### CODE REVIEW

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/couchbase-monitoring-integration.mdx [hunk 1]
```diff
@@ -114,30 +115,34 @@ To install the Couchbase integration, follow the instructions for your environme
...
+- If enabled via Amazon ECS: see [Monitor services running on ECS](/docs/integrations/host-integrations/host-integrations-list/monitor-services-running-amazon-ecs).
+- If installed on-host: edit the config in the integration's YAML config file, `couchbase-config.yml`.
 
 An integration's YAML-format configuration is where you can place required login credentials and configure how data is collected. Which options you change depend on your setup and preference.
 
 The configuration file has common settings applicable to all integrations, such as `interval`, `timeout`, `inventory_source`. To read all about these common settings, refer to our [Configuration Format](/docs/create-integrations/infrastructure-integrations-sdk/specifications/host-integrations-newer-configuration-format/#configuration-basics) document.
 
 <Callout variant="important">
-  If you are still using our legacy configuration/definition files, please refer to this [document](/docs/create-integrations/infrastructure-integrations-sdk/specifications/host-integrations-standard-configuration-format/) for help.
+  If you are still using our legacy configuration/definition files, please refer
```
Comment (nr-kkenney) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/couchbase-monitoring-integration.mdx:142]: was this line purposefully split?

Comment (roadlittledawn) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/couchbase-monitoring-integration.mdx:142]: i think that was my prettier maybe?

Comment (roadlittledawn) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/couchbase-monitoring-integration.mdx:142]: yeah it was. not sure why it wanted to break the line there. i can undo and push up ¯\_(ツ)_/¯

Comment (roadlittledawn) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/couchbase-monitoring-integration.mdx:142]: it did some weird stuff with table elements that had empty lines between them too. it added single white space

**Review by nr-kkenney — APPROVED** [review]

**Comment by nr-kkenney** [comment]
there are a couple awkward looking markdown links being split across two lines, but the site build, and i looked at a couple pages and they worked so im cool since thats what matters.

**Pull request merged**